backend/src/sms/services/sms-template.service.ts

```diff
@@ -149,8 +153,17 @@ const storeTemplate = async ({
   campaignId,
   body,
 }: StoreTemplateInput): Promise<StoreTemplateOutput> => {
+  if (!client.stripTags(body).trim()) {
+    throw new TemplateError('Message template cannot be empty!')
```

Comment:
If you enter `<foo></foo>`. The error thrown will still be `Message template cannot be empty`. Might be confusing for user since it's not technically empty.

Reply:
Yep was wondering how to better the error to be more user-friendly. How about `Message template cannot be empty or only contain HTML tags with empty content`.

Reply:
I think that works!